apps/api/src/index.js

```diff
@@ -51,19 +50,25 @@ app.use(bodyParser());
 app.use(error(formatError));
 app.use(compress());
 
+if (config.https) {
```

Comment:
Would it be simpler to use  `NODE_ENV === 'production'` instead of adding a new environment variable ?

Reply:
I'm really against checking on environment on the app
But I checked that it's default to true, so on prod it will be ok :)